src/frontend/components/SchemaForm/_RenderFormInput.tsx

```diff
@@ -70,10 +71,37 @@ export function RenderFormInput(props: IRenderFormInputProps) {
 
   switch (type) {
     case "email":
-    case "password":
     case "url":
     case "color":
       return <FormInput type={type} {...formProps} />;
+    case "password":
+    return (
```

Comment:
Can move this to a new component, `FormPasswordInput`

Reply:
Okay sir